catalog/ui/catalog-ui-search/src/main/webapp/component/query-status/query-status-header.hbs

```diff
@@ -13,25 +13,13 @@
  --}}
 <th>
     Source
-    <div class="column-absolute">
-    Source
-    </div>
 </th>
-<th>
+<th data-help="This is the number of results that made it into the Top.">
```

Comment:
Technically it isn't the number returned though.  It's the number returned that were relevant enough (based on sorting) to stick around.